src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
-  margin: 0;
-}
+html {
+  --spec-color: #00ACDC;
+
+  font-family: Roboto, sans-serif;
+  }
+
+  body {
+    margin: 0;
+    padding: 0;
+    }
+
+    .header {
+      align-items: center;
+      justify-content: space-between;
+      margin: 0;
+      padding: 0 50px;
+      display: flex;
+      }
+
+      .a {
+       text-decoration: none;
+       font-size: 0;
+       }
+
+      .img {
+        width: 40px;
+        height: 40px;
+        display: block;
+        }
+
+      .nav {
+        display: block;
+      }
+
+      .nav_list {
+       margin: 0;
+       padding: 0;
+       display: flex;
+       }
+
+      .nav_item {
+       list-style: none;
+       }
+
+      .nav_list :not(:first-child) {
+        margin-left: 20px;
+       }
+
+      .nav_link {
+        color: #000;
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 500;
+        text-transform: uppercase;
+        text-decoration: none;
+        height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead